jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/MemberScope.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 VicTools.
+ * Copyright 2020 VicTools & Sascha Kohlmann
```

Comment:
Why change the copyright?
This class was originally created in 2019 and while Sascha Kohlmann has provided valuable feedback on multiple occasions and ported the Javax module into the Jakarta module, I don't remember him touching his class here at any time.